validator/discord/discord.go

```diff
@@ -0,0 +1,118 @@
+package discord
+
+import (
+	"bufio"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"github.com/bwmarrin/discordgo"
+	"github.com/nextdotid/proof-server/config"
+	"github.com/nextdotid/proof-server/util"
+	"golang.org/x/xerrors"
+	"net/url"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/nextdotid/proof-server/types"
+	"github.com/nextdotid/proof-server/util/crypto"
+	"github.com/nextdotid/proof-server/validator"
+)
+
+type Discord struct {
+	*validator.Base
+}
+
+var (
+	re            = regexp.MustCompile(MATCH_TEMPLATE)
+	POST_TEMPLATE = map[string]string{
+		"default": "Verifying my discord ID: %s on NextID. \nSig: %%SIG_BASE64%%",
+		"en-US":   "Verifying my discord ID: %s on NextID. \nSig: %%SIG_BASE64%%",
+		"zh-CN":   "在NextID上认证我的账号:%s。\nSig: %%SIG_BASE64%%",
```

Comment:
Space missing before and after `%s`

Reply:
<img width="624" alt="image" src="https://user-images.githubusercontent.com/13994404/162139079-2ba00a30-cc9b-4646-a4c4-25dd3e2b7891.png">
still there

Reply:
`zh-CN` version.